scripts/system_tests_utils.ts

```diff
@@ -52,166 +47,29 @@ export const getEnvVar = (name: string): string | undefined =>
 	// eslint-disable-next-line @typescript-eslint/no-unsafe-return, @typescript-eslint/no-unsafe-call, @typescript-eslint/no-unsafe-member-access
 	global.Cypress ? Cypress.env(name) : process.env[name];
 
-export const DEFAULT_SYSTEM_PROVIDER = 'http://localhost:8545';
+export const DEFAULT_SYSTEM_PROVIDER = 'http://127.0.0.1:8545';
```

Comment:
was localhost an issue?

Reply:
yes. because I map 127.0.0.1 to SSL certificate